ui-admin/src/forms/designer/NewQuestionForm.tsx

```diff
@@ -64,10 +97,12 @@ export const NewQuestionForm = (props: NewQuestionFormProps) => {
 
         <Button
           variant="primary"
+          disabled={readOnly || !selectedQuestionType || !questionStableId}
```

Comment:
Not directly related to this PR but now you can't save a question with a blank stable ID. More validation coming in the future